src/ServiceConnections/RestServiceConnection.cpp

```diff
@@ -163,6 +153,19 @@ Result<void> RestServiceConnection::SendJpegPreviewImage(
 #pragma endregion
 
 #pragma region Private methods
+std::unique_ptr<httplib::Client> RestServiceConnection::getHttpClientWithAuth() {
+    auto httpClient = std::make_unique<httplib::Client>(hostname, port);
+    if (this->authToken.length() > 0)
+    {
+        httplib::Headers headers
+        {
+            {"Authorization", authToken}
```

Comment:
It's a little weird we take the whole authorization header value here but call it an auth "token", usually I'd use the word token specifically for bearer token type authorization headers.

But that's a minor quibble and I don't think it's worth changing the behavior right now.

Reply:
True true